src/Mod/Draft/draftutils/init_draft_statusbar.py

```diff
@@ -0,0 +1,182 @@
+"""Draft Statusbar commands.
+
+This module provide the code for the Draft Statusbar, activated by initGui
+"""
+## @package init_tools
```

Comment:
This is needed for Doxygen.
```
## @package init_draft_statusbar
```

Reply:
oops, was it a commit? I'll add them anyway :) thx

Reply:
Well, if you don't add them it's not a problem, it's just that in this case, you copied the string from the `init_tools.py` module, so it is documenting the wrong module. If you add the string, you should use the proper module name, which in this case is `init_draft_statusbar`.